src/providers/vod/vod-provider.ts

```diff
@@ -381,7 +381,7 @@ export class VodProvider extends Provider {
   }
 
   private _filterCuePointsOutOfVideoRange(cuePoints: any[], seekFrom: number, clipTo: number): any[] {
-    return cuePoints.filter((cp: any) => cp.startTime >= seekFrom && (cp.endTime <= clipTo || cp.endTime === Number.MAX_SAFE_INTEGER));
+    return cuePoints.filter((cp: any) => cp.startTime >= seekFrom && cp.startTime < clipTo);
```

Comment:
does changes works for live stream when endTime of cues can be `Number.MAX_SAFE_INTEGER` ?

Reply:
I changed the logic of filtering the cuepoints, no longer taking `Number.MAX_SAFE_INTEGER` into consideration.